client/src/components/Form/Elements/FormCheck.vue

```diff
@@ -0,0 +1,36 @@
+<script setup>
+import { computed, defineProps, defineEmits } from "vue";
```

Comment:
```suggestion
import { computed } from "vue";
```
I think you can omit these 2 imports. As per the [documentation](https://vuejs.org/api/sfc-script-setup.html#defineprops-defineemits):
```markdown
defineProps and defineEmits are compiler macros only usable inside <script setup>.
They do not need to be imported, and are compiled away when <script setup> is processed.
```